Sources/RemoteNotification.swift

```diff
@@ -108,119 +136,26 @@ public struct RemoteNotification: CustomStringConvertible, Equatable {
                 return
             }
             
-            body = alert[alertBodyKey] as? String
-            bodyLocalizationKey = alert[alertBodyLocKey] as? String
-            bodyLocalizationArguments = alert[alertBodyLocArgsKey] as? [String]
+            body = alert[alertBodyKey] as! String?
+            bodyLocalizationKey = alert[alertBodyLocKey] as! String?
+            bodyLocalizationArguments = alert[alertBodyLocArgsKey] as! [String]?
```

Comment:
Weird isn't necessarily a bad thing. Let's bring discussion of this up to https://github.com/square/SuperDelegate/pull/2/files/644fa7d3cd436cec2047214a885802d97eaf2466#r78670959, though. I'm a recent convert to the `as! Type?` practice and would like more perspectives.